### PR TITLE
Parameterise ELB Health Check

### PR DESCRIPTION
**Pull request by lawliet89**

### CODE REVIEW

File: modules/core/README.md [hunk 1]
```diff
@@ -443,3 +443,126 @@ aws autoscaling \
...
+| vault_asg_name | Name of the Autoscaling group for Vault cluster |
+| vault_cluster_default_user_data | Default launch configuration user data for Vault Cluster |
+| vault_cluster_size | Number of instances in the Vault cluster |
+| vault_iam_role_arn | IAM Role ARN for Vault |
+| vault_iam_role_id | IAM Role ID for Vault |
+| vault_launch_config_name | Name of the Launch Configuration for Vault cluster |
+| vault_s3_bucket_arn | ARN of the S3 bucket that Vault's state is stored |
+| vault_security_group_id | ID of the Security Group for Vault |
+| vault_servers_cluster_tag_key | Key that Vault instances are tagged with |
+| vault_servers_cluster_tag_value | Value that VAult instances are tagged with |
```
Comment (guangie88) [inline modules/core/README.md]: `VAult`

File: modules/core/variables.tf [hunk 2]
```diff
@@ -305,13 +305,87 @@ variable "vault_user_data" {
...
-  description = "Time before an unhealthy Elastic Load Balancer target becomes removed"
+variable "nomad_server_lb_deregistration_delay" {
+  description = "The time to wait for in-flight requests to complete while deregistering a target. During this time, the state of the target is draining."
+  default     = 30
+}
+
+variable "nomad_server_lb_healthy_threshold" {
+  description = "The number of consecutive health checks successes required before considering an unhealthy target healthy (2-10)."
+  default     = 2
+}
```
Comment (guangie88) [inline modules/core/variables.tf]: The value has been changed from 5 to 2 after refactoring into a variable, is this intentional?

Comment (lawliet89) [reply modules/core/variables.tf]: Yes. 5 is too long for a service recovery.

File: modules/core/outputs.tf [hunk 3]
```diff
@@ -94,39 +114,48 @@ output "vault_api_address" {
...
 output "vault_security_group_id" {
-  value = "${module.vault.security_group_id}"
+  description = "ID of the Security Group for Vault"
+  value       = "${module.vault.security_group_id}"
 }
 
 output "vault_s3_bucket_arn" {
-  value = "${module.vault.s3_bucket_arn}}"
+  description = "ARN of the S3 bucket that Vault's state is stored"
+  value       = "${module.vault.s3_bucket_arn}}"
```
Comment (guangie88) [inline modules/core/outputs.tf]: There is an extra `}` at the end in the value.

File: modules/core/variables.tf (hunk 2 above)
Comment (guangie88) [reply modules/core/variables.tf]: Okay noted

**Pull request merged**